### PR TITLE
Added modern theme for chevron component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme)

**Review by Blackbaud-TrevorBurch — DISMISSED** [review]
Looks good from a code standpoint. Just need @Blackbaud-AdamFunderburk or @Blackbaud-ToddRoberts to sign off.

File: src/app/public/modules/chevron/chevron.component.scss [hunk 1]
```diff
@@ -55,3 +55,34 @@
...
 }
+
+@include sky-theme-modern {
+  .sky-chevron {
+    height: 19px;
+    width: 16px;
+  }
+
+  .sky-chevron-part {
+    background: darken($sky-text-color-icon-borderless, 20%);
```
Comment (Blackbaud-ToddRoberts) [inline src/app/public/modules/chevron/chevron.component.scss:66]: where did this darken come from vs using the borderless icon color straight up?

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/chevron/chevron.component.scss:66]: Oops, that was a mistake. Before I make the change...what should this be? Just `$sky-text-color-icon-borderless;` ?

Comment (Blackbaud-ToddRoberts) [reply src/app/public/modules/chevron/chevron.component.scss]: i believe that is correct. it should function as if it was a borderless icon button

Comment (Blackbaud-ToddRoberts) [reply src/app/public/modules/chevron/chevron.component.scss]: which, we also need to make sure it has the correct hover, focus and active states toward that end

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme)

**Comment by Blackbaud-AdamFunderburk** [comment]
image
😰

**Comment by Blackbaud-AdamFunderburk** [comment]
> image
> 😰

This looks like a distant edge case. Not worth sweating over.

File: src/app/public/modules/chevron/chevron.component.scss (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/chevron/chevron.component.scss]: @Blackbaud-ToddRoberts updated to act like a borderless icon button in modern theme. Should have the focus/hover/etc states.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-chevron-modern-theme)

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-AlexKingman - I know that I am totally splitting hairs. However, I noticed that the chevron appears shifted down from center when it is in the active state. I verified this in paint and am getting it being roughly 7px from top and 5px from bottom. This isn't worth holding it up since it is one pixel either way, but I figured it was worth mentioning.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Comment by Blackbaud-AlexKingman** [comment]
> @Blackbaud-AlexKingman - I noticed that the chevron appears shifted down from center when it is in the active state.

We were trying to match the borderless-icon button. If you look at a chevron button using those styles in modern theme, you'll see its also shifted down a bit.
Screen Shot 2021-03-17 at 9 26 20 AM

**Pull request merged**